research/src/pages/tabs.proposal.mdx

```diff
@@ -9,8 +9,7 @@ showInMenu: false
 # Tabs Component Specification
 
 ## Overview <a href="#overview" id="overview"></a>
-
-*Tabs* are a set of sections of content that display one panel of content at a time. Each tab panel has an associated tab element, that when activated, displays the panel. The list of tab elements is arranged along one edge of the currently displayed panel.
+*Tabs* are a component which provides [well-defined meaning and interaction](https://www.w3.org/TR/wai-aria-practices/#tabpanel) allowing one-at-a-time, non-sequential viewing over a series of thematically grouped, state independent and labelled sections. They merely increase the users ability to understand and consume these bits of content.
```

Comment:
"They merely increase the users ability to understand and consume these bits of content." <-- I prefer not to use terms like "merely". This sentence doesn't seem to help define tabs.